### ISSUE TITLE
fail2ban service does not start

### ISSUE THREAD

**Issue by tangentsoft** [Homebrew/homebrew-core#98944]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
## brew config

HOMEBREW_VERSION: 3.4.5
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 33709297cf9cb37605cf4efcb7151905baf8ccf5
Last commit: 8 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: d1303bc2a422d4d85c8eb9c8dc2d4e3ab67ab379
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.kluhC42HpD/org.xquartz:0
HOMEBREW_EDITOR: mvim -v
HOMEBREW_MAKE_JOBS: 20
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: 20-core 64-bit cometlake
Clang: 13.1.6 build 1316
Git: 2.32.0 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.3-x86_64
CLT: 13.3.0.0.1.1645755326
Xcode: N/A


## brew doctor

Your system is ready to brew.
```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Get fail2ban running in the background so I can have it operate on some logs I have.

### What happened (include all command output)?

I believe I've boiled the failure down to this simple test case:

```
% fail2ban-python /usr/local/opt/fail2ban/libexec/bin/fail2ban-server
2022-04-09 19:12:12,062 fail2ban                [17660]: ERROR   cannot import name 'Mapping' from 'collections' (/usr/local/Cellar/python@3.10/3.10.2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/collections/__init__.py)
```

You can also see it without digging into the "opt" tree with commands like this one:

```
% fail2ban-regex /path/to/my.log 'login failure'
Traceback (most recent call last):
  File "/usr/local/Cellar/fail2ban/0.11.2_1/libexec/bin/fail2ban-regex", line 32, in <module>
    from fail2ban.client.fail2banregex import exec_command_line
  File "/usr/local/Cellar/fail2ban/0.11.2_1/libexec/lib/python3.10/site-packages/fail2ban/client/fail2banregex.py", line 55, in <module>
    from ..server.filter import Filter, FileContainer
  File "/usr/local/Cellar/fail2ban/0.11.2_1/libexec/lib/python3.10/site-packages/fail2ban/server/filter.py", line 33, in <module>
    from .actions import Actions
  File "/usr/local/Cellar/fail2ban/0.11.2_1/libexec/lib/python3.10/site-packages/fail2ban/server/actions.py", line 31, in <module>
    from collections import Mapping
ImportError: cannot import name 'Mapping' from 'collections' (/usr/local/Cellar/python@3.10/3.10.2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/collections/__init__.py)
```

I believe this is a manifestation of [a closed upstream bug](https://github.com/fail2ban/fail2ban/issues/2610). I'd take this up there except that it hasn't been reopened now that Python 3.10 is released as was proposed, indicating lack of interest in fixing it upstream. Also, at the end of that issue's comment chain Fedora says they're carrying a patch to fix it, so that might be the "solution", such as it is.

### What did you expect to happen?

This should work:

```
% sudo brew services start fail2ban
```

It claims to succeed, and if you "list" the services immediately after, it appears to have started, but several seconds later you get 

```
% brew services list
Name     Status       User    File
fail2ban error  65280 root    ~/Library/LaunchAgents/homebrew.mxcl.fail2ban.plist
```

I believe this is due to the same cause as the lower-level test cases above.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
% brew install fail2ban
% fail2ban-python /usr/local/opt/fail2ban/libexec/bin/fail2ban-server
```

**Comment by cho-m** [comment]
Should be fixed with #98959.

**Comment by tangentsoft** [comment]
Verified. Thank you!
